test/unit/suite.spec.js

```diff
@@ -493,6 +493,16 @@ describe('Suite', function() {
         new Suite('Bdd suite', 'root');
       }, 'not to throw');
     });
+
+    it('should be root suite', function() {
+      var rootSuite = new Suite('', {}, true);
+      expect(rootSuite.root, 'to be', true);
+    });
+
+    it('should not be root suite', function() {
+      var suite = new Suite('');
+      expect(suite.root, 'to be', false);
+    });
```

Comment:
Maybe it's just me, but these two tests don't really add much value since they just verify your ability to set a property correctly (which admittedly has been done wrong in the past). Wouldn't a more meaningful test be to verify that only a single Suite can have the `root` property set `true`?